packages/textkit/src/attributedString/runAt.js

```diff
@@ -10,7 +10,7 @@ import runIndexAt from './runIndexAt';
  * @return {Object} run
  */
 const runAt = (n, attributedString) => {
-  const runIndex = runIndexAt(n)(attributedString);
+  const runIndex = R.max(runIndexAt(n)(attributedString))(null);
```

Comment:
This was needed because `runIndexAt` can return `-1`, and `R.path` now supports negative indices, so instead of returning `undefined` it would return the last element.

https://github.com/ramda/ramda/pull/2670

This change seems to be a bit controversial as [it was reported as a bug](https://github.com/ramda/ramda/issues/2974) and there's [an open PR to revert it](https://github.com/ramda/ramda/pull/3116), but that's been open for over a year now so it seems like we should roll with the change.